examplec.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN"
```

Comment:
This PR contains a number of files you used for testing.

- `examplec.html`
- `exampleflex.html`
- `pygments/lexers/aainput.txt`
- `pygments/lexers/out.txt`
- `test.c`
- `test.exe`

They should not be part of the main repository; please remove them from this PR (`git rm --cached examplec.html exampleflex.html ...; git commit`; this leaves them untouched in your working tree, they just disappear from what will eventually be merged).

Reply:
@erickhartawan I see a commit from you entitled "[Removed local testing files](https://github.com/pygments/pygments/pull/2134/commits/0f1f927265258a0b47bbd36ee5625e5661b2f064)", but it doesn't actually remove the files.

Reply:
I think I made a mistake. will get it fixed soon!